### PR TITLE
L1 multi-asset stablecoin blockchain Stative Network

### PR DESCRIPTION
**Pull request by soxocx**
#### Project Abstract

The Stative Network will be a L1 multi-asset stablecoin blockchain.

The vision of Stative is to create a base layer for a truly decentralized monetary system for web3.

Based on Stative dApps will be able to easily build web3-native applications for exchange and store of value, using stable decentralized abstractions of real world currencies. A decentralized web3 world bank.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/grant_guidelines_per_category.md).
- [x] A BTC, Ethereum (USDT/USDC/DAI) or Polkadot/Kusama (aUSD) address for the payment of the milestones is provided inside the application.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Review by semuelle — CHANGES_REQUESTED** [review]
Thank you for this thorough application, @soxocx! Someone will look into it soon. In the meantime, could you revise the following:

- All `0` deliverables still contain the template text as specification. Can you at least specify a license (0a) and the type (and medium) of article you plan to publish (0e)?
- What exactly is being delivered (and how) in 2a.1, 2a.2a and 3?
- I assume all the `(ETH)` contract deliverables will be written in Solidity, which we generally don't fund. In that case, you might have to remove or replace those deliverables.
- Will the `(STA)` contracts be written in _ink!_?

Also, the deliverables in 2d are partly not specific enough. However, the whole milestone is so specific to your situation/setup that I doubt it can get funded by the grants program. I'll get a second opinion on that.

**Comment by semuelle** [comment]
Hi @soxocx. Thanks for the update. 
Regarding 2d, I'd say the legal opinion and blog post might be of broader use. Corporate structure and token design are usually not things we care about. Anyway, I'll share your application with the rest of the committee, so they can chime in. 

> Should we still squash/force to have a single commit in this pr just update the application?

No need to squash during review. We'll squash 'em anyway on merge.

**Comment by alxs** [comment]
Hi @soxocx, thank you for your application. I agree with Sebastian's comments regarding the last milestone, but it also applies to the whole application to some extent. I'd like to make you aware of the following section in our [guidelines](https://github.com/w3f/Grants-Program#guidelines):
> Applications must not mention a specific token. Furthermore, the focus of the application should lie on the software that is being implemented/research being carried out as part of the grant, and less on your project/venture/operation. For the purpose of the application and delivery, think about how others might also benefit from your work.

Could you please adapt your application so that it meets these requirements? Also, could you list the milestones starting from 1? This would simplify recordkeeping on our end.

**Review by takahser — COMMENTED** [review]
@soxocx thanks for submitting your grant application.
I have a couple of questions, after skimming your doc:

- Have you looked into Statemint/Statemine and their support for stablecoins? How would your platform compare to that?

- As indicated by other W3F grants committee members, we don't support any research regarding [your own token](https://github.com/w3f/Grants-Program/pull/1304/files#diff-0feefc7c40444646fd3a05065f3d6cabd6e8df552052af6ca0694dcb621f07b7R298). Feel free to remove any sections that refer to anything related to token design/tokenomics, etc.

> The governance token Stative "STA" can be minted by natively swapping against the NSTs.

1. I'm not clear about how you'd determine how many units of the specific NST (e.g. USDT) has to be locked to mint 1 STA. Could you elaborate on that?
2. For the sake of simplicity, let's assume, we'd lock 1 USDT to mint 1 STA using the price determined in 1). Immediately after that, we'd lock 1 EURT to mint another 1 STA, assuming USDEUR is at 1.00 EUR at that time, making the total locked collateral worth 2 USD (ignoring any USDT and EURT de-pegging risk). Now, let's assume USDEUR surges 10% to 1.10 EUR 1d later. We're now in a situation where the collateral is worth less than 2 USD. Can I now go ahead and burn my 1 STA for 1 USDT and my other 1 STA for 1 EURT?
3. If the answer for 2) is yes, I think STA isn't fungible, since they can be exchanged for NSTs that are worth different amounts.

**Review by dsm-w3f — COMMENTED** [review]
@soxocx thank you for your application. What is the long-term roadmap for this project? I mean, this is the first of how many grant applications you plan to do. Do you plan to fund-raise it for other fonts? Could you share the long-term vision to complete this project and launch it?

File: applications/stative_network.md
Comment (Noc2) [inline applications/stative_network.md:246]: What do you mean by Connect Chainlinks parachain? It's probably better to focus only on the integration of the chainlink module on a local chain.  Btw. did you take a look at the code by Acala, and how will your solution be different regarding the use of oracle data?

File: applications/stative_network.md
Comment (Noc2) [inline applications/stative_network.md:264]: Do you want to develop your own bridge here, or does this leverage an existing solution? ([Snowbridge](https://github.com/Snowfork/snowbridge) for example?)

Comment (kkug3l) [reply applications/stative_network.md:264]: Yes, here we are planning to develop our own bridges. Main reason is the fact, that we - again - have specific requirements due to the concept of aggregated sets backing the NSTs and governance token. Our bridges 
a. need to be able to lock different tokens as part of the aggregated sets that are backing the NSTs and
b. need to be able to lock NSTs (so a native token of the stative parachain) as part of an aggregated set that is backing the governance token
For bridges to enable minting wrapped tokens for the native tokens of our stative parachain, we will definitely check leveraging existing solutions already working with ETH and the like.
Last but not least due to the recent developments and incidents in the context of bridges, we feel the need to build a solution from zero with max focus on security and a very thorough auditing.

**Comment by alxs** [comment]
@soxocx we recently updated our T&C's, would you mind signing the CLA (first comment) again?

File: applications/stative_network.md
Comment (Noc2) [reply applications/stative_network.md:242]: Could you still remove the connection to the chainlink parachain from the delivery here? (There is no chainlink parachain)

File: applications/stative_network.md
Comment (Noc2) [reply applications/stative_network.md:260]: In this case, could you provide more information about your bridge? I personally see a huge risk in setting up your own independent chain rather than simply using a parachain slot, at least in the beginning. Implementing a decentralized bridge is a very difficult task.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the update. But I had some follow-up questions regarding my previous comments.

**Comment by kkug3l** [comment]
hey guys, kindly following up to check if you might have additional feedback or questions for us?

**Review by Noc2 — CHANGES_REQUESTED** [review]
Sorry for the late reply. Do you have some detailed technical documentation or something similar regarding the design of your bridge? For example, is it a multisig solution or a fully trustless bridge that implements light clients, etc? Are you going to use Merkle Mountain Ranges or something similar, etc?

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the update. In this case, it’s a little bit hard for me to evaluate the application or at least the bridge-related deliveries. If you indeed need to develop your own bridge, I would still recommend using [Snowbridge](https://github.com/Snowfork/snowbridge) and potentially changing it a little bit rather than “build a solution from zero.” I think it makes more sense to leverage the existing technology here and potentially contribute to it.

**Comment by kkug3l** [comment]
@Noc2 Thanks for your feedback - I think in term of efficiency it would be great if you could help us with some guidance in concluding the application process. 

My understanding is, that with the iterations on the application and the additional input we met your general requirements. The remaining issue for you in order to do a final evaluation is the bridge part of the application, correct? 

As @soxocx stated, we understand your concern, but consider building a bride specific to the needs of stative a vital part of the project. We are convinced, that such a bridge (as part of the stative network) would be a great benefit to establish and expand a striving stablecoin ecosystem within Polkadot. Of course we are also positive, that we are up to the task in doing so. In the process of working on that task, we will of course research all options in achieving this goal - including building on existing solutions like Snowbridge. But since the requirements for stative are specific and the topic of bridges is as you said yourself a very sensitive one, it is from out POV vital to be in control and know every little detail from scratch and be able to adapt if needed (this might obviously be a disadvantage when building on an existing solution). This means we would use the funds from the grant from that milestone also on the evaluation of the best way in achieving the goal of implementing the best solution possible for the task.

Please let me point out, that we are highly motivated to build stative, as it is in our opinion a highly innovative concept with the potential of achieving high adoption and a thriving community during the next wave. 

We would love to kick-start the project with this grant in order get things going and in doing so contribute to the Polkadot ecosystem by collaborating and building re-usable tech. The stative network will be a decentral, open-source project committed to a long-term cooperation with the Polkadot community - so the grant would really create a win-win situation.

Any feedback helping us in achieving this goal efficiently is highly appreciated - meaning, if there are specific things we need to change in order to get a clear yes / no answer on the grant, it would be great if you could share them with us 🙏

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the reply. In general, the grant application is ready for review, which means that any of the grants committee members can take a look at the application and approve it if they think it’s a good project at any time. 

However, I personally won’t approve it at the moment. Your deliveries currently say “Substrate smart contract (Ink): NST-Swap-Brdige” without actually technically specifying how this bridge is implemented, and it’s also not clear to me why we need to support this additional bridge or at least the benefits of it. Keep in mind that we usually don’t care so much about the business model, market impact, etc, about a specific project, but we mostly care about the technical deliverables.

**Comment by Noc2** [comment]
Thanks again for the application and all the effort you put into this. Given that the rest of the committee also isn’t willing to support it, I will close the PR. Let me know if you have any questions.

**Pull request closed without merging**
